server/routerlicious/packages/services-telemetry/src/schema.ts

```diff
@@ -27,23 +27,27 @@ export abstract class BaseLumberjackSchemaValidator implements ILumberjackSchema
     }
 
     // Validators
-    protected readonly idValidation = (propValue) => {
+    protected readonly idValidation = (propValue: any) => {
         return this.isUndefined(propValue)
-            || (this.checkType(propValue, "string") && propValue.length > 0);
+            || (this.isString(propValue) && propValue.length > 0);
     };
 
-    protected readonly seqNumberValidation = (propValue) => {
+    protected readonly seqNumberValidation = (propValue: any) => {
         return this.isUndefined(propValue)
-            || (this.checkType(propValue, "number") && propValue >= -1);
+            || (this.isNumber(propValue) && propValue >= -1);
     };
 
     // Helpers
-    private isUndefined(propValue: any) {
+    private isUndefined(propValue: any): propValue is undefined {
```

Comment:
Should these also be `protected` such that extensions can reference these helpers? Or maybe `protected static`?

Reply:
I think the idea is that the helpers be used only in the validators provided in `BaseLumberjackSchemaValidator` - the validators are `protected`. That's because my idea was to "force" extensions to use the validators instead.